test/NodejsEmbeddingTests.cs

```diff
@@ -19,28 +19,49 @@ namespace Microsoft.JavaScript.NodeApi.Test;
 
 public class NodejsEmbeddingTests
 {
+    private static string MainScript { get; } =
+        "globalThis.require = require('module').createRequire(process.execPath);\n";
+
     private static string LibnodePath { get; } = GetLibnodePath();
 
     // The Node.js platform may only be initialized once per process.
-    internal static NodejsPlatform? NodejsPlatform { get; } =
-        File.Exists(LibnodePath) ? new(LibnodePath, args: new[] { "node", "--expose-gc" }) : null;
+    internal static NodeEmbeddingPlatform NodejsPlatform { get; } =
+        new(LibnodePath, new NodeEmbeddingPlatformSettings
+        {
+            Args = new[] { "node", "--expose-gc" }
+        });
 
-    internal static NodejsEnvironment CreateNodejsEnvironment()
+    internal static NodeEmbeddingThreadRuntime CreateNodejsEnvironment()
```

Comment:
Update the "Environment" test method names to the new "ThreadRuntime" terminology.

Reply:
Renamed to `CreateNodeEmbeddingThreadRuntime`